modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractDartCodegen.java

```diff
@@ -271,8 +279,10 @@ public void processOpts() {
         }
 
         if (additionalProperties.containsKey(CodegenConstants.SOURCE_FOLDER)) {
-            this.setSourceFolder((String) additionalProperties.get(CodegenConstants.SOURCE_FOLDER));
+            String srcFolder = (String) additionalProperties.get(CodegenConstants.SOURCE_FOLDER);
+            this.setSourceFolder(srcFolder.replace('/', File.separatorChar));
         }
+        additionalProperties.put(CodegenConstants.SOURCE_FOLDER, sourceFolder);
```

Comment:
is this null safe?

Reply:
I think so, I have not found a way to pass `null` via a property (file/CLI) - it is always an empty String